blog/content/post/2020-10-19-security-part-2.md

```diff
@@ -0,0 +1,97 @@
+
+Let's say we have already paid the penalty costs. Now we have to make up for these losses somehow, but will our application still be able to earn for itself? If the leak was serious and third parties suffered as a result, there will be a loss of reputation, and hence the loss of regular and new customers. You also have to count on the attack of the competition, which will certainly emphasize the favorable situation for them, as our clients will also learn.
+
+Finally, there will be problems such as our ranking in Google, which will have a negative impact on sales very quickly.
+Google is constantly investigating whether we use best security practices such as TLS / SSL, which are used to encrypt traffic and ensure the confidentiality and integrity of data transmission, as well as server and sometimes customer authentication.
+
+The bottom line is that there is no argument that would challenge making security a priority. Disregarding it does not pay off, and the seemingly saved costs may bring about the opposite result than intended.
+
+### How to deal with security challenges?
+
+The first step is the right approach to the software development process.
+
+There are some rules to follow:
+
+- continuous development of your skills through your own development and training within the organization,
+- sharing knowledge about safety in the team,
+- taking responsibility for not releasing an underdeveloped product under time pressure,
+- good understanding of the impact of risk on the business aspects of operations,
+- following external guidelines such a OWASP,
+- primarily treating safety as priority, not a necessary evil
+
+The rules are obvious, but how many cases do we really stick to them? In how many cases are we not looking for the best solutions or we do not pay enough attention to tests due to the race against time?
+
+We have to answer these questions ourselves. Being aware of your own approach is the beginning of the road to trying to combine speed and security when developing software.
+
+### What's next? Know your enemy.
+
+Fortunately, the problems we face are similar for all developers and there are organizations that work to describe most of them and teach others how to protect themselves from outside threats. Therefore, we do not have to reinvent the wheel.
+
+##### OWASP (Open Web Application Security Project)
+
+is a nonprofit foundation that works to improve the security of software. [OWASP page](https://owasp.org/)
+
+This organization publishes a list of the most common errors in web applications. This list was established in 2017 and is based on real data obtained from companies, organizations and professionals involved in security testing.
+
+##### The current list for 2020:
+
+- Injection
```

Comment:
Probably a short description next to these vulnerabilities might be a good idea.